src/hooks/useEngineConnectionSubscriptions.ts

```diff
@@ -164,20 +164,49 @@ export function useEngineConnectionSubscriptions() {
               if (artifact?.type === 'plane') {
                 const planeInfo =
                   await sceneEntitiesManager.getFaceDetails(planeOrFaceId)
+
+                // Apply camera-based orientation logic similar to default planes
+                let zAxis: [number, number, number] = [
+                  planeInfo.z_axis.x,
+                  planeInfo.z_axis.y,
+                  planeInfo.z_axis.z,
+                ]
+                let yAxis: [number, number, number] = [
+                  planeInfo.y_axis.x,
+                  planeInfo.y_axis.y,
+                  planeInfo.y_axis.z,
+                ]
+
+                // Get camera vector to determine which side of the plane we're viewing from
+                const camVector = sceneInfra.camControls.camera.position
+                  .clone()
+                  .sub(sceneInfra.camControls.target)
+
+                // Determine the canonical (absolute) plane orientation
+                const absZAxis: [number, number, number] = [
+                  Math.abs(zAxis[0]),
+                  Math.abs(zAxis[1]),
+                  Math.abs(zAxis[2]),
+                ]
+
+                // Find the dominant axis (like default planes do)
+                const maxComponent = Math.max(...absZAxis)
+                const dominantAxisIndex = absZAxis.indexOf(maxComponent)
+
+                // Check camera position against canonical orientation (like default planes)
+                const cameraComponents = [camVector.x, camVector.y, camVector.z]
+                let negated = cameraComponents[dominantAxisIndex] < 0
```

Comment:
We can tell which side of the plane they are on if the cameraComponent along the normal of the plane is < 0 or not.